omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,103 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
+
+Group setup
+-----------
+
+#. Create a dedicated :doc:`public group <public>` to host the raw data
+   underlying the publication.
+#. Add all the authors of the paper to this new public group.
+#. If you have not already done so, configure the :ref:`public_user` and add
```

Comment:
split this step into two steps. 

  - If you have not already done so, configure the :ref:`public_user`
  - Add the public user as a member of the newly created read-only group